libs/ui/src/components/MediaItem/MediaItem.vue

```diff
@@ -11,138 +11,137 @@
       :is-detail="isDetail" />
     <div
       v-if="isLewd && isLewdBlurredLayer"
-      class="nsfw-blur is-flex is-align-items-center is-justify-content-center is-flex-direction-column">
+      class="nsfw-blur is-capitalized is-flex is-align-items-center is-justify-content-center is-flex-direction-column">
       <NeoIcon icon="eye-slash" class="mb-3" />
-      <span class="heading-nsfw"> {{ $t('lewd.explicit') }} </span>
-      <span class="nsfw-desc">{{ $t('lewd.explicitDesc') }}</span>
-      <span class="nsfw-content" @click="showContent()">{{
-        $t('lewd.showContent')
+      <span class="nsfw-heading has-text-weight-bold">
+        {{ $t('lewd.explicit') }}
+      </span>
+      <span class="nsfw-desc text-align-center">{{
+        $t('lewd.explicitDesc')
       }}</span>
     </div>
+    <NeoButton
+      v-if="isLewd"
+      rounded
+      no-shadow
+      class="nsfw-action px-4 py-1 is-size-6"
+      :class="[isLewdBlurredLayer ? '' : 'hide']"
+      :label="
+        isLewdBlurredLayer ? $t('lewd.showContent') : $t('lewd.hideContent')
+      "
+      @click.native="toggleContent" />
   </div>
 </template>
 
-<script lang="ts">
-import { defineAsyncComponent } from 'vue'
-
+<script lang="ts" setup>
 import { getMimeType, resolveMedia } from '@/utils/gallery/media'
-import NeoIcon from './../NeoIcon/NeoIcon.vue'
+import { NeoButton, NeoIcon } from '@kodadot1/brick'
+import ImageMedia from './type/ImageMedia.vue'
+import VideoMedia from './type/VideoMedia.vue'
+import AudioMedia from './type/AudioMedia.vue'
+import ModelMedia from './type/ModelMedia.vue'
+import JsonMedia from './type/JsonMedia.vue'
+import IFrameMedia from './type/IFrameMedia.vue'
+import ObjectMedia from './type/ObjectMedia.vue'
+import Media from './type/UnknownMedia.vue'
 
 const SUFFIX = 'Media'
+const props = withDefaults(
+  defineProps<{
+    src?: string
+    animationSrc?: string
+    mimeType?: string
+    title?: string
+    original?: boolean
+    isLewd?: boolean
+    isDetail?: boolean
+    placeholder?: string
+  }>(),
+  {
+    src: '',
+    animationSrc: '',
+    mimeType: 'image/png',
+    title: 'KodaDot NFT',
+    original: false,
+    isLewd: false,
+    isDetail: false,
+    placeholder: '',
+  }
+)
+// props.mimeType may be empty string
+let mimeType = ref(!!props.mimeType ? props.mimeType : 'image/png')
```

Comment:
```suggestion
const mimeType = ref(props.mimeType)
```

1. we very rarely use let. always const
2. since `props.mimeType` already defaults to `'image/png'` there is no need for this extra step

Reply:
i found `prop.mimeType` may pass empty string ""  so  add `image/png` as fallback